_aggregations/metric/median-absolute-deviation.md

```diff
@@ -9,70 +9,37 @@ redirect_from:
 
 # Median absolute deviation aggregations
 
-The `median_absolute_deviation` metric is a single-value metric aggregation that returns a median absolute deviation field. Median absolute deviation is a statistical measure of data variability. Because the median absolute deviation measures dispersion from the median, it provides a more robust measure of variability that is less affected by outliers in a dataset. 
+The `median_absolute_deviation` metric is a single-value metric. Median absolute deviation is a variability metric that measures dispersion from the median.
 
-Median absolute deviation is calculated as follows:<br>
-median_absolute_deviation = median(|X<sub>i</sub> - Median(X<sub>i</sub>)|)
+Median absolute deviation is less affected by outliers than standard deviation, which relies on squared error terms, and is useful for describing data that is not normally distributed.
 
-The following example calculates the median absolute deviation of the `DistanceMiles` field in the sample dataset `opensearch_dashboards_sample_data_flights`:
+Median absolute deviation is computed as follows:<br/>
+median_absolute_deviation = median( | X<sub>i</sub> - Median(X<sub>i</sub>) | )
 
+In practice, for large datasets `median_absolute_deviation` is estimated. You can adjust the trade-off between estimation accuracy and performance; see [Adjusting estimation accuracy](#adjusting-estimation-accuracy).
```

Comment:
"In practice, for large datasets `median_absolute_deviation` is estimated."

I'm not sure why do we need to add this detail as major statistical metrics are only useful when evaluated upon a large dataset. Plus, there is no quantification of what exactly `large` is.

----

Or alternatively, did you mean something like: 


In practice, estimating `median_absolute_deviation` for large data_sets is computationally expensive.  You can adjust the trade-off between estimation accuracy and performance; see [Adjusting estimation accuracy](#adjusting-estimation-accuracy).

(Actually I like this explanation, thoughts?)

Reply:
You're right, "large" data sets are ill-defined here. As I understand it (correct me if I'm wrong), the key factor is whether the dataset fits in memory, and is complicated by doing comparisons across multiple nodes in map-reduce. 

I'd argue that the statistic is useful for _relatively_ small datasets (say hundreds or a few thousand) that could be computed exactly, but I take your point that in many (most?) cases the dataset is potentially massive.

```suggestion
In practice, `median_absolute_deviation` is estimated. This estimation is computationally expensive. You can adjust the trade-off between estimation accuracy and performance; see [Adjusting estimation accuracy](#adjusting-estimation-accuracy) for details.
```

Reply:
I think your suggestion makes sense. 

I want to add up a small clarification as well in the first statement:

In practical implementations, `median_absolute_deviation` is estimated rather than calculated exactly due to memory restrictions. This estimation is computationally expensive. You can adjust the trade-off between estimation accuracy and performance; see [Adjusting estimation accuracy](#adjusting-estimation-accuracy) for details.

---- 

Adding that detail since the steps involved in calculating the MAD are exact. Its the memory implications which prevent us from calculating the exact MAD and that's why we rely on estimation algorithms like TDigest here (explained later).

Reply:
> In OpenSearch, irrespective of the data size, we follow the same TDigest algortihm.

Thanks, that's important to know. 

```suggestion
OpenSearch estimates `median_absolute_deviation` rather than calculating it directly due to memory limitations. This estimation is computationally expensive. You can adjust the trade-off between estimation accuracy and performance; see [Adjusting estimation accuracy](https://github.com/opensearch-project/documentation-website/pull/9453/files#adjusting-estimation-accuracy) for details.
```